virt/release_notes/virt-4-19-release-notes.adoc

```diff
@@ -156,82 +114,40 @@ Some features in this release are currently in Technology Preview. These experim
 
 [discrete]
 [id="virt-4-18-ki-nodes"]
-==== Nodes
-//CNV-38543 - 4.16 Still an issue
-* Uninstalling {VirtProductName} does not remove the `feature.node.kubevirt.io` node labels created by {VirtProductName}. You must remove the labels manually. (link:https://issues.redhat.com/browse/CNV-38543[*CNV-38543*])
+=== Nodes
 
-//BZ 2151169
-* In a heterogeneous cluster with different compute nodes, virtual machines that have HyperV reenlightenment enabled cannot be scheduled on nodes that do not support timestamp-counter scaling (TSC) or have the appropriate TSC frequency. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2151169[*BZ#2151169*])
 
 [discrete]
 [id="virt-4-18-ki-storage"]
-==== Storage
-//CNV-23501: 4.16 - Keep per Dominik Holler
-* If you clone more than 100 VMs using the `csi-clone` cloning strategy, then the Ceph CSI might not purge the clones. Manually deleting the clones might also fail. (link:https://issues.redhat.com/browse/CNV-23501[*CNV-23501*])
-** As a workaround, you can restart the `ceph-mgr` to purge the VM clones.
-
-// CNV-55104: 4.18 - Unresolved
-* If you perform storage class migration for a stopped VM, the VM might not be able to start because of a missing bootable device. To prevent this, do not attempt storage class migration if the VM is not running. (link:https://issues.redhat.com/browse/CNV-55104[*CNV-55104*])
-+
---
-:FeatureName: Storage class migration
-include::snippets/technology-preview.adoc[]
---
+=== Storage
+
 
 [discrete]
 [id="virt-4-18-ki-virtualization"]
-==== Virtualization
-
-//CNV-36448: 4.16 - Unresolved
-* When adding a virtual Trusted Platform Module (vTPM) device to a Windows VM, the BitLocker Drive Encryption system check passes even if the vTPM device is not persistent. This is because a vTPM device that is not persistent stores and recovers encryption keys using ephemeral storage for the lifetime of the `virt-launcher` pod. When the VM migrates or is shut down and restarts, the vTPM data is lost. (link:https://issues.redhat.com/browse/CNV-36448[*CNV-36448*])
+=== Virtualization
 
-//CNV-33835: 4.16 - Unresolved
-* {VirtProductName} links a service account token in use by a pod to that specific pod. {VirtProductName} implements a service account volume by creating a disk image that contains a token. If you migrate a VM, then the service account volume becomes invalid. (link:https://issues.redhat.com/browse/CNV-33835[*CNV-33835*])
-** As a workaround, use user accounts rather than service accounts because user account tokens are not bound to a specific pod.
 
 [discrete]
 [id="virt-4-16-ki-webconsole_{context}"]
-==== Web console
-//CNV-38594: 4.16 - Still an issue per Guohua
-* When you create a persistent volume claim (PVC) by selecting *With Data upload form* from the *Create PersistentVolumeClaim* list in the web console, uploading data to the PVC by using the *Upload Data* field fails. (link:https://issues.redhat.com/browse/CNV-37607[*CNV-37607*])
+=== Web console
 
 [discrete]
```

Comment:
Should lines 220 and 221 be commented out?

Reply:
@lahinson  No, nothing should be commented out (except some instructions) until the release notes are ready. If some sections turn out to be empty in the end, then their headings and [discrete] lines will be commented out. I've removed the commenting everywhere and changed the version number in the heading IDs - please take a look.